### PR TITLE
[Bug] Update the way addresses are presented in PDMO

### PR DESCRIPTION
**Pull request by chibujax**
# JIRA TICKET NAME:
PDMO-34: [Change the way addresses are presented in PDMO](https://issues.openmrs.org/browse/PDMO-34)

## SUMMARY:
Update the way addresses are presented in PDMO using the configured template

### CODE REVIEW

**Comment by Ewanjiru** [comment]
looks good.

**Comment by CeciliaWanjiku** [comment]
I see no problems.. its good

**Review by jacksono — COMMENTED** [review]
Changes  and code look good

**Review by Ewanjiru — COMMENTED** [review]
Since the three files are for the same component, will it be a good idea to put the in a folder?

**Comment by CeciliaWanjiku** [comment]
Also the add new button..could we bring it below the address?

**Comment by jacksono** [comment]
We had also talked about changing the way a preferred attribute is displayed without necessarily having to use a preferred check box or boolean, did you consider any alternative ways?

**Comment by chibujax** [comment]
@Ewanjiru Done

**Comment by chibujax** [comment]
@CeciliaWanjiku We have a method already followed as shown in identifiers. But all the same, I have tried to add the button under addresses card but it looks weird(pictures attached). Please note that addresses are displayed as columns. So if one address card is displayed, the add new card follows on the next column as a two column-table display. Remember a patient can have multiple addresses and having the add button at the end of display may not look nice (Check what we did with names). What I have done is to bring the add card to front, so that even with multiple address cards, the add card is conspicuous
screen shot 2017-09-19 at 11 16 56 am
screen shot 2017-09-19 at 11 20 38 am
screen shot 2017-09-19 at 11 16 56 am
screen shot 2017-09-19 at 11 20 47 am
screen shot 2017-09-19 at 11 23 18 am




.

**Comment by chibujax** [comment]
@jacksono We have a ticket that handles "preferred" and "soft deleted" for our project, would it be wise to single it out here alone? Besides, the ticket is not handling preferred display but address display. What do you think?

**Comment by CeciliaWanjiku** [comment]
@andela-cuchendu It is worth noting @Ewanjiru is working to redesign how identifiers look.

**Comment by jacksono** [comment]
@andela-cuchendu I think thats fine, they will be handled separately then.

File: app/js/components/subAddress/addressForm.jsx [hunk 1]
```diff
@@ -163,36 +114,16 @@ export default class AddressForm extends React.Component {
...
- * delete - deletes address from database
- *
- * @param {object} e - event object
- * @param {string} uuid - address uuid
- *
- * @memberOf AddressForm
- */
   delete(e,uuid){
     this.setState({isShowingModal: false});
     apiCall(this.state.editValues, 'delete', `person/${this.state.parentUuid}/address/${this.state.uuid}?!purge`)
```
Comment (dkayiwa) [inline app/js/components/subAddress/addressForm.jsx]: This being purging, do we have support for voiding?

**Comment by chibujax** [comment]
Oh! Thank you for pointing that out. 'Purge' was added when we were tying to find out why we could not retrieve deleted OBS, so I even tried 'delete' from the wiki which is different from the swagger doc and forgot to revert. Thank you, corrected now.

**Pull request merged**